.circleci/cimodel/data/simple/android_gradle.py

```diff
@@ -1,37 +0,0 @@
-import cimodel.data.simple.util.branch_filters
-
-
-class AndroidJob:
-    def __init__(self, job_name, template_name, dependencies):
-        self.job_name = job_name
-        self.template_name = template_name
-        self.dependencies = dependencies
-
-    def gen_tree(self):
-
-        props_dict = {
-            "filters": cimodel.data.simple.util.branch_filters.gen_filter_dict(),
-            "name": self.job_name,
-            "requires": self.dependencies,
-        }
-
-        return [{self.template_name: props_dict}]
-
-
-WORKFLOW_DATA = [
-    AndroidJob(
-        "pytorch-linux-xenial-py3-clang5-android-ndk-r19c-gradle-build-x86_32",
```

Comment:
The current code sets master branch filter on android gradle jobs unconditionally - I think this job should not have master branch filter. It's supposed to run on PR but not run on master commits: https://github.com/pytorch/pytorch/blob/master/.circleci/verbatim-sources/job-specs/job-specs-custom.yml#L284
With the current config the job doesn't run at all. In this PR I added `is_master_only=False` for this job. If we disabled the job on PR for good reason (too flaky?) then we should consider deleting it.